prisma/migrations/20240317225208_db_schema_v1/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "user" (
```

Comment:
It is always standard practice to name your tables in plural as it is a collection of records
```suggestion
CREATE TABLE "users" (
```